AtomicLongStableReadStressTest.testChangingCluster

```
java.lang.AssertionError: com.hazelcast.client.stress.AtomicLongStableReadStressTest$StressThread1 encountered an error expected null, but was:<com.hazelcast.core.HazelcastInstanceNotActiveException: Hazelcast instance is not ready yet!>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotNull(Assert.java:664)
	at org.junit.Assert.assertNull(Assert.java:646)
	at com.hazelcast.client.stress.StressTestSupport$TestThread.assertNoError(StressTestSupport.java:171)
	at com.hazelcast.client.stress.StressTestSupport.assertNoErrors(StressTestSupport.java:122)
	at com.hazelcast.client.stress.StressTestSupport.joinAll(StressTestSupport.java:143)
	at com.hazelcast.client.stress.AtomicLongStableReadStressTest.test(AtomicLongStableReadStressTest.java:71)
	at com.hazelcast.client.stress.AtomicLongStableReadStressTest.testChangingCluster(AtomicLongStableReadStressTest.java:59)
```
https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.maintenance-nightly/309/com.hazelcast$hazelcast-client/testReport/com.hazelcast.client.stress/AtomicLongStableReadStressTest/testChangingCluster/

testChangingCluster
```
java.lang.AssertionError: com.hazelcast.client.stress.AtomicLongStableReadStressTest$StressThread1 encountered an error expected null, but was:<com.hazelcast.core.HazelcastInstanceNotActiveException: State: PASSIVE Operation: class com.hazelcast.concurrent.atomiclong.operations.GetOperation>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotNull(Assert.java:755)
	at org.junit.Assert.assertNull(Assert.java:737)
	at com.hazelcast.client.stress.StressTestSupport$TestThread.assertNoError(StressTestSupport.java:171)
	at com.hazelcast.client.stress.StressTestSupport.assertNoErrors(StressTestSupport.java:122)
	at com.hazelcast.client.stress.StressTestSupport.joinAll(StressTestSupport.java:143)
	at com.hazelcast.client.stress.AtomicLongStableReadStressTest.test(AtomicLongStableReadStressTest.java:72)
	at com.hazelcast.client.stress.AtomicLongStableReadStressTest.testChangingCluster(AtomicLongStableReadStressTest.java:60)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-nightly/com.hazelcast$hazelcast-client/994/testReport/junit/com.hazelcast.client.stress/AtomicLongStableReadStressTest/testChangingCluster/

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.x-nightly/994/#showFailuresLink

--- a/hazelcast-client/src/test/java/com/hazelcast/client/stress/MapStableReadStressTest.java
+++ b/hazelcast-client/src/test/java/com/hazelcast/client/stress/MapStableReadStressTest.java
@@ -56,7 +56,7 @@ public class MapStableReadStressTest extends StressTestSupport {
         }
     }
 
-    //@Test
+    @Test
     public void testChangingCluster() {
         test(true);
     }

--- a/hazelcast/src/main/java/com/hazelcast/concurrent/atomiclong/operations/GetOperation.java
+++ b/hazelcast/src/main/java/com/hazelcast/concurrent/atomiclong/operations/GetOperation.java
@@ -18,8 +18,9 @@ package com.hazelcast.concurrent.atomiclong.operations;
 
 import com.hazelcast.concurrent.atomiclong.AtomicLongContainer;
 import com.hazelcast.concurrent.atomiclong.AtomicLongDataSerializerHook;
+import com.hazelcast.spi.ReadonlyOperation;
 
-public class GetOperation extends AbstractAtomicLongOperation {
+public class GetOperation extends AbstractAtomicLongOperation implements ReadonlyOperation {
 
     private long returnValue;
 

--- a/hazelcast/src/main/java/com/hazelcast/concurrent/atomicreference/operations/GetOperation.java
+++ b/hazelcast/src/main/java/com/hazelcast/concurrent/atomicreference/operations/GetOperation.java
@@ -18,8 +18,9 @@ package com.hazelcast.concurrent.atomicreference.operations;
 
 import com.hazelcast.concurrent.atomicreference.AtomicReferenceDataSerializerHook;
 import com.hazelcast.nio.serialization.Data;
+import com.hazelcast.spi.ReadonlyOperation;
 
-public class GetOperation extends AbstractAtomicReferenceOperation {
+public class GetOperation extends AbstractAtomicReferenceOperation implements ReadonlyOperation {
 
     private Data returnValue;
 

--- a/hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/operation/GetOperation.java
+++ b/hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/operation/GetOperation.java
@@ -24,10 +24,11 @@ import com.hazelcast.replicatedmap.impl.ReplicatedMapService;
 import com.hazelcast.replicatedmap.impl.record.ReplicatedRecord;
 import com.hazelcast.replicatedmap.impl.record.ReplicatedRecordStore;
 import com.hazelcast.spi.AbstractOperation;
+import com.hazelcast.spi.ReadonlyOperation;
 
 import java.io.IOException;
 
-public class GetOperation extends AbstractOperation implements IdentifiedDataSerializable {
+public class GetOperation extends AbstractOperation implements IdentifiedDataSerializable, ReadonlyOperation {
 
     private String name;
     private Data key;

--- a/hazelcast/src/main/java/com/hazelcast/spi/ReadonlyOperation.java
+++ b/hazelcast/src/main/java/com/hazelcast/spi/ReadonlyOperation.java
@@ -19,9 +19,7 @@ package com.hazelcast.spi;
 import com.hazelcast.spi.impl.AllowedDuringPassiveState;
 
 /**
- * Read-only operations are not retried during migration.
- *
- * @author mdogan 7/17/13
+ * Read-only operations are allowed to run during migration and passive state.
  */
 public interface ReadonlyOperation extends AllowedDuringPassiveState {
 

